packages/suite-desktop/src-electron/modules/auto-updater.ts

```diff
@@ -187,6 +199,12 @@ const init = ({ mainWindow, store }: Dependencies) => {
                     'auto-updater',
                     `Is configured to auto update after app quit ${shouldInstallUpdateOnQuit}`,
                 );
+
+                // save current app version so that after app is relaunch we can show info about transition to the new version
+                store.setUpdateSettings({
+                    ...store.getUpdateSettings(),
+                    preUpdateVersion: app.getVersion(),
+                });
```

Comment:
I'm wondering what stops us from doing this within the init step where we also dispatch the toast notification. It would require a bit less indirection and the logic would be confined to a single place.

The meaning of the `preUpdateVersion` variable would have to change slightly to something like `currentVersion` as it would be kept 1:1 with the actual version most of the time except the first start after an update.

In other words, I propose: let's have a `currentVersion` stored in the store. On every app start compare it with the app's actual version and if they differ dispatch a notification and update the store to the actual version.

What do you think?

Reply:
I was considering this option as well but decided to avoid having variable `currentVersion` in store because it feels very confusing. Imagine you invetigate `store.ts` later and see `currentVersion` there. You would say: why the hell are we saving something we already know here? So thats why I decided to go with some name that clearly states its purpose and its value is set only during the update routine. That much for my motivation. 
Your solution has advantage of solving everything in one place. I don't know, should I rework it?

Reply:
Understand. Let's keep it as is. 👍